libs/openFrameworksCompiled/project/android/build.gradle

```diff
@@ -49,6 +55,15 @@ task ofNdkSetup {
     /* Find suitable make executable */
     def make = null;
 
+    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+        /* Require MinGW on Windows, edit paths to use forward slashes */
+        make = "C:/MinGW/msys/1.0/bin/make.exe"
```

Comment:
this assumes a certain install location for MinGW - can't we make this more flexible?